README.md

```diff
@@ -123,6 +123,18 @@ To enable full control over behave's behaviour (if you will...), the `-i <ini fi
 
 For a full inventory of tests available to run, use the `./manage tests`. Note that tests in the list tagged @wip are works in progress and should generally not be run.
 
+## Using AATH Agents as Services
```

Comment:
It would be really nice to add a concrete example (or point to one) about how this can be used.  It's pretty hard to figure out why this is being added without that.  I'm sure there is a good reason, but I'm not sure what it is.

Reply:
Added some use cases for the feature. Let me know if this is clear.